From c63a675642ad993bbf5547e09d3192e4e35dfe6b Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 19 Mar 2014 09:40:31 +0800
Subject: [PATCH 302/304] ENGR00303478 Build: The build name display wrong

Update the BUILD_ID to beta rc3 for next build_id

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/build_id.mk       | 2 +-
 arm2_6sl/build_id.mk       | 2 +-
 evk_6sl/build_id.mk        | 2 +-
 hdmidongle_6dq/build_id.mk | 2 +-
 imx5x/build_id.mk          | 2 +-
 sabreauto_6q/build_id.mk   | 2 +-
 sabresd_6dq/build_id.mk    | 2 +-
 7 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/arm2_6dq/build_id.mk b/arm2_6dq/build_id.mk
index d3bbc7a..04ea60c 100644
--- a/arm2_6dq/build_id.mk
+++ b/arm2_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc1
+export BUILD_ID=1.0.0-beta-rc3
 export BUILD_NUMBER=20140418
diff --git a/arm2_6sl/build_id.mk b/arm2_6sl/build_id.mk
index d3bbc7a..04ea60c 100644
--- a/arm2_6sl/build_id.mk
+++ b/arm2_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc1
+export BUILD_ID=1.0.0-beta-rc3
 export BUILD_NUMBER=20140418
diff --git a/evk_6sl/build_id.mk b/evk_6sl/build_id.mk
index d3bbc7a..04ea60c 100755
--- a/evk_6sl/build_id.mk
+++ b/evk_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc1
+export BUILD_ID=1.0.0-beta-rc3
 export BUILD_NUMBER=20140418
diff --git a/hdmidongle_6dq/build_id.mk b/hdmidongle_6dq/build_id.mk
index d3bbc7a..04ea60c 100644
--- a/hdmidongle_6dq/build_id.mk
+++ b/hdmidongle_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc1
+export BUILD_ID=1.0.0-beta-rc3
 export BUILD_NUMBER=20140418
diff --git a/imx5x/build_id.mk b/imx5x/build_id.mk
index d3bbc7a..04ea60c 100644
--- a/imx5x/build_id.mk
+++ b/imx5x/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc1
+export BUILD_ID=1.0.0-beta-rc3
 export BUILD_NUMBER=20140418
diff --git a/sabreauto_6q/build_id.mk b/sabreauto_6q/build_id.mk
index d3bbc7a..04ea60c 100644
--- a/sabreauto_6q/build_id.mk
+++ b/sabreauto_6q/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc1
+export BUILD_ID=1.0.0-beta-rc3
 export BUILD_NUMBER=20140418
diff --git a/sabresd_6dq/build_id.mk b/sabresd_6dq/build_id.mk
index d3bbc7a..04ea60c 100644
--- a/sabresd_6dq/build_id.mk
+++ b/sabresd_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc1
+export BUILD_ID=1.0.0-beta-rc3
 export BUILD_NUMBER=20140418
-- 
1.8.0

